packages/ui/src/theme/types.ts

```diff
@@ -125,7 +125,7 @@ export interface DefaultTheme extends Pick<Theme, 'name' | 'overrides'> {
  * to be used in Javascript/Typescript.
  */
 export interface WebTheme
-  extends Pick<Theme, 'breakpoints' | 'name' | 'overrides'> {
+  extends Pick<DefaultTheme, 'breakpoints' | 'name' | 'overrides'> {
```

Comment:
This is actually the correct type because a WebTheme will always have all the `breakpoints`